python-sdk/src/astro/sql/operators/cleanup.py

```diff
@@ -18,6 +17,10 @@
 from astro.sql.operators.load_file import LoadFileOperator
 from astro.sql.table import Table
 
+if TYPE_CHECKING:
+    # TODO: This can be removed from TYPE_CHECKING once there is a minimum requirement of Airflow 2.2.3+
+    from airflow.utils.context import Context
```

Comment:
Let's make sure to change to `"Context"` in all places of the current file as you have done it for the other file.